README.md

```diff
@@ -28,81 +28,110 @@ The website is built using **Next.js**, a React-based framework for server-side
 
 **Vercel** is used for deployment, providing a fast and scalable hosting environment for the website. Once a pull request is made, Vercel will automatically deploy a staged version of the website for final approval prior to deploying these changes to the public-facing website. Learn more about Vercel [here](https://vercel.com/docs).
 
-**Supabase** is used for Postgres database services and authentication with Google accounts via OAuth. 
+**Supabase** is used for Postgres database services and authentication with Google and Azure accounts via OAuth. 
 
 
 ## System Requirements
 
+Install:
 1. [Node.js](https://nodejs.org/en/)
-2. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. [VS Code](https://code.visualstudio.com/download)
+1. [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
 
 ## Local Environment Setup
 
-1. Setup the frontend environment
+The application requires setup of both a Next.JS frontend and a Supabase backend.
+
+### Setup the frontend environment.
+    
+1. Go in the folder you want the project to exist, run:
 
    ```bash
    git clone https://github.com/openseattle/seattle-humane-app
-   cd seattle-humane-app
-   git checkout main
-   npm install
    ```
 
-2. Setup VSCode for auto-formatting
+1. Open VS code.
+
+1. From VS code, open the command palette using `⇧⌘P` and type:
```

Comment:
Also include the Windows shortcut ctrl+shift+P?

Reply:
Good call! I'll create a new PR with this